fleetshard/pkg/central/reconciler/reconciler.go

```diff
@@ -453,12 +452,20 @@ func (r *CentralReconciler) ensureChartResourcesExist(ctx context.Context, remot
 		out.SetGroupVersionKind(obj.GroupVersionKind())
 		err := r.client.Get(ctx, key, &out)
 		if err == nil {
+			glog.Infof("Updating object %s/%s", obj.GetNamespace(), obj.GetName())
+			obj.SetResourceVersion(out.GetResourceVersion())
+			err := r.client.Update(ctx, obj)
+			if err != nil {
+				return fmt.Errorf("failed to update object %s/%s of type %v: %w", key.Namespace, key.Namespace, obj.GroupVersionKind(), err)
+			}
+
 			continue
 		}
 		if !apiErrors.IsNotFound(err) {
 			return fmt.Errorf("failed to retrieve object %s/%s of type %v: %w", key.Namespace, key.Name, obj.GroupVersionKind(), err)
 		}
 		err = r.client.Create(ctx, obj)
+		glog.Infof("Creating object %s/%s", obj.GetNamespace(), obj.GetName())
```

Comment:
Reducing verbosity in production systems by setting `V(10)`:

```suggestion
		glog.v(10).Infof("Creating object %s/%s", obj.GetNamespace(), obj.GetName())
```